R/add_CI.R

```diff
@@ -105,12 +107,12 @@ add_CI.ggsurvfit <- function(gg,
     gg <- gg +
       ggplot2::geom_ribbon(ggplot2::aes(ymin = est.lower, 
                                         ymax = est.upper, 
-                                        fill = "transparent", 
-                                        colour = strata),
-                           alpha = alpha, 
+                                        fill = strata, 
+                                        colour = strata,
+                                        alpha = alpha),
                            outline.type = "both",
                            linetype = linetype, 
-                           show.legend = FALSE,
+                           show.legend = TRUE,
```

Comment:
should be FALSE. Nobody wants to see color around the linetype in a clinical setting. We can discuss and make this an optional argument, but TRUE should not be the default.

Reply:
Both, `TRUE` and `FALSE` actually don't have an influence on the outcome here so I'll remove it.